job-runner.toml

```diff
@@ -0,0 +1,37 @@
+# Install to ~/.config/cockpit-dev/job-runner.toml
+
+[container]
+command = 'podman'
+args = []
+run-args = ['--device=/dev/kvm']
+default-image = 'quay.io/cockpit/tasks:latest'
+
+[container.secrets]
+# see podman-secret(1)
+# github-token = '--secret=github-token'
+
+[logs]
+driver='local'   # 's3' or 'local'
+
+[logs.s3]
+# hint: podman run --rm --net=host quay.io/minio/minio server /var
+url = 'http://127.0.0.1:9000/tmp/'
+key = {access='minioadmin', secret='minioadmin'}
+user-agent = 'job-runner (cockpit-project/bots)'
```

Comment:
This seems uninteresting, can this become/stay an internal default? And commented out here if you care?

Reply:
See above notes about internal defaults.